src/CSVReader.java

```diff
@@ -18,7 +18,7 @@ public void run(String path) {
 
         final String CSV_File = path;
         BufferedReader br = null;
-        final String SEPARATOR = ",";
+        final String separator = ",";
```

Comment:
Sorry for confusing you....
A constant is upper case, otherwise it is lower/camelcase 

A constant in java is declare as : **final static**
Example: private final static SEPARATOR = ","

Here you do want a constant. It will never change.